### PR TITLE
Update adapter and api-augment to match 34ba6b42bef658f9963f3f2e4cc34ff07857a26e

### PR DESCRIPTION
**Pull request by imstar15**
- Fix adapter for the change of config package.
- Fix code for eslint

### CODE REVIEW

File: packages/adapter/src/chains/chainData.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  ss58Prefix: number | undefined;
+
+  name: string | undefined;
+
+  xcmInstructionNetworkType: XcmInstructionNetworkType =
+    XcmInstructionNetworkType.Null;
+
+  xcm: XcmConfig | undefined;
+}
```
Comment (chrisli30) [inline packages/adapter/src/chains/chainData.ts:27]: This is already defined in `/packages/config/src/chains/types/Chain.ts`. Please import the type from config package.

Besides, the `network` attribute now lives within `XcmConfig`.

File: packages/sdk/src/index.ts [hunk 2]
```diff
@@ -1,21 +1,32 @@
-import _ from 'lodash';
-import BN from 'bn.js';
-import type { SubmittableExtrinsic } from '@polkadot/api/types';
-import { u8aToHex } from '@polkadot/util';
-import type { HexString } from '@polkadot/util/types';
-import type { KeyringPair } from '@polkadot/keyring/types';
-import type { ApiPromise } from '@polkadot/api';
-import { ChainAdapter, OakAdapter, TaskSchedulerChainAdapter, SendExtrinsicResult, XcmInstructionNetworkType } from '@oak-network/adapter';
-import { Weight } from '@oak-network/sdk-types';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (chrisli30) [inline packages/sdk/src/index.ts:1]: Please DO NOT blindly add this `eslint-disable` without understanding its cause. Reference: 
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md

These libraries are not defined in the `dependencies` block of `package.json`! Please fix the [package.json](https://github.com/OAK-Foundation/oak.js/blob/dev/packages/sdk/package.json) and remove these code comments!

**Review by chrisli30 — CHANGES_REQUESTED** [review]
Thank you for fixing the adapter, but there are still problems in code.

File: packages/sdk/src/index.ts (hunk 2 above)
Comment (imstar15) [reply packages/sdk/src/index.ts]: Fixed

File: packages/adapter/src/chains/chainData.ts (hunk 1 above)
Comment (imstar15) [reply packages/adapter/src/chains/chainData.ts:27]: Yes, eslint requires that we can't have two classes in the same file, so I moved it out.

**Review by chrisli30 — APPROVED** [review]
The code looks good, but why you are updaing api-augment in this PR "Fix adapter"?

Please do the following,
1. Update the PR title to "Update adapter and api-augment to match <commit_hash>", in order to add the commit hash of OAK-blockchain. **What’s the version of blockchain this PR is for?**
2. Do `test/functional` all pass?

**Comment by imstar15** [comment]
> 2. Do `test/functional` all pass?

No, I have not run the tests.

**Pull request merged**